jurry/amqp-symfony-bundle/1.0/config/packages/jurry_rabbitmq.yaml

```diff
@@ -0,0 +1,9 @@
+amqp_handler:
+    connection: '%env(RABBIT_MQ_DSN)%'
```

Comment:
You might want to add something like a vendor prefix, e.g. `JURRY_RABBITMQ_DSN`, to prevent clashes with other bundles.

Reply:
Prefix added